css-view-transitions-1/Overview.bs

```diff
@@ -893,15 +893,21 @@ The {{ViewTransition/domUpdated}} [=getter steps=] are to return [=this's=] [=Vi
     To <dfn>update transition DOM</dfn> given a {{ViewTransition}} |transition|:
 
     1. Let |document| be |transition|'s [=relevant global object's=] [=associated document=].
-
-    1. For each [=page-transition pseudo-elements=] associated with |transition|,
-        check whether there is an active animation associated with this pseudo-element.
-
-        Issue: Define what active animation means here.
-
-    1. If no [=page-transition pseudo-elements=] has an active animation:
-
-        Issue: There needs to be a definition/link for "active animation".
+	
+	1. Let |hasActiveAnimations| be a boolean, initially false. For each [=page-transition pseudo-elements=]
+        associated with |transition|:
+    
+        1. Let |element| be the [=page-transition pseudo-element=].
+    
+        1. For each |animation| with [=effect target=] set to |element| and [=animation timeline=]
+            of type [=document timeline=], set |hasActiveAnimations| to true if any of the following
+            conditions is true:
```

Comment:
Animation's don't have an effect target. That's a property of keyframe effects.

I think we could use wording similar to that for `getAnimations()`, e.g.

```bikeshed
1. For each |animation| that contains at least one [=animation effect=]
   whose [=effect target=] is |element| and
   whose associated [=animation timeline=] is of type [=document timeline=], ...
```

Reply:
Done.